libs/sdk/testing/src/lib/matchers/matchers.ts

```diff
@@ -769,6 +769,8 @@ export function expect<T>(actual: T): SkyMatchers<T> {
  * Create an async expectation for a spec.
  * @param actual Actual computed value to test expectations against.
  */
-export function expectAsync<T, U>(actual: T): SkyAsyncMatchers<T, U> {
+export function expectAsync<T, U>(
+  actual: T | PromiseLike<T>,
```

Comment:
I was seeing a strange error with our harness assertions when `expectAsync` was imported into the same spec. Turns out, Jasmine has a slightly different implementation of its types, so I copied that over to our `expectAsync` to satisfy the type checker.